element-hq/element-x-android
# Reply on redacted and UTD rendering

## Type of change

- [x] Other : complete feature

## Content


Improve rendering of replied event when this one is redacted or cannot be decrypted

## Motivation and context


Closes #2318 

## Screenshots / GIFs



See recorded ones (and ignore screenshot files renaming...)

## Tests



- Send a message A
- Send a reply to message B
- Redact message A
- observe rendering of message B. Before: empty content. Now: "Message removed" with a picto.

There will be the same rendering with UTD message. To test:

Login to a session and from another client with access to e2e history), reply to an old message in an e2e room.

<img width="257" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/0aec34aa-83ce-4d27-aa41-59f4272a202f">

## Tested devices

- OS version(s):

## Checklist
